cli/lib/config.js

```diff
@@ -2,35 +2,37 @@
 
 module.exports = { load, save };
 
-const { promises: fs } = require('fs');
+const fs = require('fs');
 const toml = require('@iarna/toml');
 const home = require('user-home');
 const path = require('path');
 
 const errors = require('./errors');
 
 async function load(filename = path.join(home, '.entropicrc')) {
-  var content = null;
-  try {
-    content = await fs.readFile(filename, 'utf8');
-  } catch (e) {
-    if (e.code === 'ENOENT') {
-      return {};
-    }
-
-    throw new errors.CouldNotReadConfigFile(filename, e);
-  }
-
-  var parsed = null;
-  try {
-    parsed = toml.parse(content);
-  } catch (e) {
-    throw new errors.CouldNotParseConfigToml(filename, e);
-  }
+  var content = '';
+  var parsed = '';
+  await fs.readFile(filename, 'utf8', (e, data) => {    
```

Comment:
I think using the fs.promises ( `require('fs').promises` ) is cleaner for this. Also, here you're `await`ing a callback.

If you need to do some specific handling for node versions, I would use [`process.version`](https://nodejs.org/api/process.html#process_process_version) and handle accordingly. 

Though, I know the docs say that Node 12 is required, so it's worth asking @chrisdickinson if this is needed.